foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/generator/SparkApplicationGenerator.java

```diff
@@ -115,20 +114,6 @@ public void generate(GenerationContext context) {
         vc.put("versionNeo4j", config.getNeo4jVersion());
         vc.put("versionAwsSdkBundle", config.getAwsSdkBundleVersion());
 
-        // Addresses a Spark bug in which the --packages argument was not respected in cluster mode (3.0.0 <= x < 3.4.0)
-        // We don't currently support Spark <3.x.x in aiSSEMBLE, but we include a check for it just in case we ever do.
```

Comment:
I: We don't support changing the Spark version from the default aiSSEMBLE version (currently 3.5.2). It's not really worth it to keep the complexity around supporting older versions since we don't allow it.  If we revisit this due to demand, the logic isn't too bad to re-introduce. (And would likely be a very small portion of the overall work that would need to be done to support multiple Spark versions.)